### PR TITLE
chore: Add test for different layouts

### PR DESCRIPTION
**Pull request by alannnc**
## What does this PR do?

- Test booking flow for weekly and column layouts.

## Requirement/Documentation

- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change

- [X] Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?

- Are there environment variables that should be set?
- What are the minimal test data to have?
- What is expected (happy path) to have (input and output)?
- Any other important info that could help to test that PR

## Mandatory Tasks

- [ ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

File: apps/web/playwright/booking-pages.e2e.ts [hunk 1]
```diff
@@ -268,3 +268,60 @@ test.describe("prefill", () => {
...
+  test.beforeEach(async ({ page, users }) => {
+    const user = await users.create();
+    await page.goto(`/${user.username}`);
+  });
+
+  test("Book on week layout", async ({ page }) => {
+    // Click first event type
+    await page.click('[data-testid="event-type-link"]');
+
+    await page.click('[data-testid="toggle-group-item-week_view"]');
```
Comment (alannnc) [inline apps/web/playwright/booking-pages.e2e.ts:282]: clicking on week view

File: apps/web/playwright/booking-pages.e2e.ts [hunk 2]
```diff
@@ -268,3 +268,60 @@ test.describe("prefill", () => {
...
+
+    await page.click('[data-testid="incrementMonth"]');
+
+    await page.locator('[data-testid="calendar-empty-cell"]').nth(0).click();
```
Comment (alannnc) [inline apps/web/playwright/booking-pages.e2e.ts:286]: clicking on first available time

File: apps/web/playwright/booking-pages.e2e.ts [hunk 3]
```diff
@@ -268,3 +268,60 @@ test.describe("prefill", () => {
...
+
+    // expect page to be booking page
+    await expect(page.locator("[data-testid=success-page]")).toBeVisible();
+  });
+
+  test("Book on column layout", async ({ page }) => {
+    // Click first event type
+    await page.click('[data-testid="event-type-link"]');
+
+    await page.click('[data-testid="toggle-group-item-column_view"]');
```
Comment (alannnc) [inline apps/web/playwright/booking-pages.e2e.ts:307]: clicking on column view toggle

File: apps/web/playwright/booking-pages.e2e.ts [hunk 4]
```diff
@@ -268,3 +268,60 @@ test.describe("prefill", () => {
...
+
+    await page.click('[data-testid="incrementMonth"]');
+
+    await page.locator('[data-testid="time"]').nth(0).click();
```
Comment (alannnc) [inline apps/web/playwright/booking-pages.e2e.ts:311]: Clicking first available time slot

File: packages/features/calendars/weeklyview/components/event/Empty.tsx [hunk 5]
```diff
@@ -96,6 +96,7 @@ function Cell({ isDisabled, topOffsetMinutes, timeSlot }: CellProps) {
       )}
       data-disabled={isDisabled}
       data-slot={timeSlot.toISOString()}
+      data-testid="calendar-empty-cell"
```
Comment (alannnc) [inline packages/features/calendars/weeklyview/components/event/Empty.tsx:99]: test id for easy access

File: packages/ui/components/form/toggleGroup/ToggleGroup.tsx [hunk 6]
```diff
@@ -49,6 +49,7 @@ export const ToggleGroup = ({ options, onValueChange, isFullWidth, ...props }: T
             <RadixToggleGroup.Item
               disabled={option.disabled}
               value={option.value}
+              data-testid={`toggle-group-item-${option.value}`}
```
Comment (alannnc) [inline packages/ui/components/form/toggleGroup/ToggleGroup.tsx:52]: test id for easy access

**Review by zomars — APPROVED** [review]
LGTM

**Pull request merged**